src/components/StructuredContent/StructuredContent.tsx

```diff
@@ -0,0 +1,162 @@
+  imageComponent?: JSX.Element;
+  /** Optional value to render as a credit for the internal `Image` component. */
+  imageCredit?: string;
+  /** Optional value to control the positioning of the internal `Image` component.
+   * Defaults to `StructuredContentImagePosition.Left`. */
+  imagePosition?: StructuredContentImagePosition;
+  /** Optional value to control the size of the internal `Image` component.
+   * Defaults to `ImageSizes.Medium`. */
+  imageSize?: ImageSizes;
+  /** Optional value that contains the path to an image. If omitted, the internal
+   * DS `Image` component will not render. */
+  imageSrc?: string;
+  /** Required value to set the text for the body content. */
+  bodyContent: string | JSX.Element;
+}
+
+/**
+ * Internal component used in the `StructuredContent` component
+ * that renders the DS `Image` component.
+ */
+function StructuredContentImage(props: ImageProps) {
+  const {
+    additionalFigureStyles,
+    additionalImageStyles,
+    additionalWrapperStyles,
+    alt,
+    component,
+    imageAspectRatio,
+    imageCaption,
+    imageCredit,
+    imageSize,
+    src,
+  } = props;
+  return (
+    <Image
+      additionalFigureStyles={additionalFigureStyles}
+      additionalImageStyles={additionalImageStyles}
+      additionalWrapperStyles={additionalWrapperStyles}
+      alt={alt}
+      component={component}
+      imageAspectRatio={imageAspectRatio}
+      imageSize={imageSize}
+      imageCaption={imageCaption}
+      imageCredit={imageCredit}
+      src={src}
+    />
+  );
+}
+
+/**
+ * The `StructuredContent` component that displays a heading, callout content,
+ * an image, and body content. All are optional except for body content.
+ */
+export default function StructuredContent(
+  props: React.PropsWithChildren<StructuredContentProps>
+) {
+  const {
+    calloutText,
+    className,
+    headingText,
+    id = generateUUID(),
+    imageAlt = "",
+    imageAspectRatio = ImageRatios.Square,
+    imageCaption,
+    imageComponent,
+    imageCredit,
+    imagePosition = StructuredContentImagePosition.Left,
+    imageSize = ImageSizes.Medium,
+    imageSrc,
+    bodyContent,
+  } = props;
+  const hasImage = imageSrc || imageComponent;
+  const hasFigureImage = imageCaption || imageCredit;
+  const styles = useMultiStyleConfig("StructuredContent", {
+    hasFigureImage,
+    imageAspectRatio,
+    imagePosition,
+  });
+  const finalBodyContent =
+    typeof bodyContent === "string" ? (
+      <div dangerouslySetInnerHTML={{ __html: bodyContent }} />
+    ) : (
+      <Box>{bodyContent}</Box>
+    );
+
+  if (hasImage && !imageAlt) {
+    console.warn(
+      "StructuredContent: `imageAlt` prop is required when using an image."
+    );
+  }
+
+  if (
+    imagePosition === StructuredContentImagePosition.Center &&
+    imageSize !== ImageSizes.Default
+  ) {
+    console.warn(
+      `StructuredContent: image size "${imageSize}" is not supported when using the center image position.`
```

Comment:
The new "additional" styles functionality that was added to `Image` seems to have provided more control over  `Image` in `StructuredContent` and the image is now resizing properly for the center position.  Which I like.  Because of that, this warning message is no longer relevant and can be removed.